apps/passport-client/public/index.hbs

```diff
@@ -4,6 +4,12 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, user-scalable=no" />
     <link rel="stylesheet" href="{{cssPath}}" />
+    <script>
+      // set background color based on path upon initial paint
+      document.getElementsByTagName("html")[0].setAttribute("class",
+      !window.location.hash || window.location.hash.match(/^#\/({{grayBgPaths}})(\?.+)?$/
+      ) ? "bg-gray" : "bg-primary");
+    </script>
```

Comment:
i'm wary of stuff like this because inevitably we're going to forget a page.

i suggest including both a list of grey and primary background colors in the `build.ts` file, and `console.error`ing in the case that the current page path is not included in the union of those two path sets. this way, our app will remind us to set a default background color for a page, and we will be less likely to forget to configure the default background color of a page.

i would also add a pretty verbose comment describing what you're doing here.